products/sle/main.pm

```diff
@@ -775,6 +775,9 @@ sub load_extra_test () {
 sub load_x11tests() {
     return unless (!get_var("INSTALLONLY") && get_var("DESKTOP") !~ /textmode|minimalx/ && !get_var("DUALBOOT") && !get_var("RESCUECD") && !get_var("HACLUSTER"));
 
+    if (get_var("PATTERNS", '') || get_var('HDD_1', '') =~ /smt/) {
```

Comment:
I just used /smt/ It matches also `PATTERNS` and I could not imagine collision with another HDD_1